src/App.tsx

```diff
@@ -34,62 +35,133 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort();
```

Comment:
It is better to use localeCompare method to sort goods correctly alphabetically

Reply:
Done